lib/compress/zstd_compress.c

```diff
@@ -1257,29 +1274,30 @@ static size_t ZSTD_resetCCtx_usingCDict(ZSTD_CCtx* cctx,
         32 KB, /* ZSTD_btopt */
         8 KB /* ZSTD_btultra */
```

Comment:
unrelated to this patch : 
I'm surprised that the cut-off value for `ZSTD_btultra` goes down to 8 KB, significantly lower than previous strategies, and especially lower than `ZSTD_btopt`, which I would expect to be pretty close.
Is that intentional ?

Reply:
I'd have to dig up the data to find the graphs that supported that choice, but yeah. My rationalization was that btultra does *so many searches* that double searching both contexts gets prohibitively expensive very quickly. I dunno.